tests/pytest/core/test_middleware_login_required.py

```diff
@@ -34,6 +34,7 @@ def test_login_auth_required(app_request, mocked_view, decorated_view):
 
 
 @pytest.mark.django_db
+@pytest.mark.usefixtures("model_EligibilityVerifier")
 def test_login_auth_not_required(app_request, mocked_view, decorated_view):
     verifier = EligibilityVerifier.objects.filter(auth_provider__isnull=True).first()
```

Comment:
inject the specific `EligibilityVerifier` needed rather than querying

Reply:
Fixed in [5e90973](https://github.com/cal-itp/benefits/pull/788/commits/5e909730fb58c86a66cc0cb438c7951e83253273)